.github/workflows/azure-terraform.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       # Validate the Terraform code using inbuilt
       # validate command
       - name: Validate Module
-        uses: docker://hashicorp/terraform:0.12.29
+        uses: docker://hashicorp/terraform:1.0.0
         with:
           entrypoint: terraform
-          args: validate cloud_Azure/terraform/module/tests/ -no-color
+          args: -chdir=cloud_Azure/terraform/module/tests/ validate -no-color
```

Comment:
I wonder why is the `-no-color` option required/desirable?

Reply:
Can't ask the author as he is no longer available; from github actions examples found on internet I see that this is a common practice to use terraform with -no-color